src/main.h

```diff
@@ -254,10 +254,19 @@ void UnlinkPrunedFiles(std::set<int>& setFilesToPrune);
 
 /** Get statistics from node state */
 bool GetNodeStateStats(NodeId nodeid, CNodeStateStats &stats);
-/** Increase a node's misbehavior score. */
-void Misbehaving(NodeId nodeid, int howmuch);
+
 /** Flush all state, indexes and buffers to disk. */
 void FlushStateToDisk();
+
+CBlockIndex* AddToBlockIndex(const CBlockHeader& block);
+
+enum FlushStateMode {
```

Comment:
If you move the enum, you probably also want to move to the header the signature of its overload;
`bool static FlushStateToDisk(CValidationState &state, FlushStateMode mode)`

Reply:
>If you move methods like Misbehaving from the global namespace to the Network namespace, its Ok to make the method start with a lowercase character and be consistent with the coding styleguide.

Love that, coming from java land I hate uppercased methods :)